setup.md

```diff
@@ -21,3 +21,12 @@
 * Each label is stuck on a post-it: yellow = doc, red = bug, green = feature.
 
 * Make some large labels to group related issues (e.g. ggplot2, r-lib) on the wall.
+
+## Icebreaker
+
+The file `bingo.pdf` with source `bingo.qmd` contains instructions for an icebreaker game to help get attendees chatting. To prepare:
+
+* Print 80 copies of the `bingo.pdf` file.
+* Purchase pens for those that don't happen to have one--50 should be enough.
```

Comment:
Feedback on this guess much appreciated before I order a pack of pens later today.😆

Reply:
Yeah, I think 50 should be plenty 😄